SetupDataPkg/Library/ActiveProfileSelectorLibNull/ActiveProfileSelectorLibNull.inf

```diff
@@ -0,0 +1,32 @@
+## @file
+# Null instance of ActiveProfileSelectorLib
+#
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION         = 0x00010017
+  BASE_NAME           = ActiveProfileSelectorLibNull
+  FILE_GUID           = 23757D9D-D703-47E1-90B5-266FD8CBF928
+  VERSION_STRING      = 1.0
+  MODULE_TYPE         = BASE
+  LIBRARY_CLASS       = ActiveProfileSelectorLib
+
+[Sources]
+  ActiveProfileSelectorLibNull.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  SetupDataPkg/SetupDataPkg.dec
+
+[LibraryClasses]
+  BaseLib
+  DebugLib
+  PcdLib
+  BaseMemoryLib
+  MemoryAllocationLib

```

Comment:
memory allocation can probably be removed now

Reply:
Thanks, removed.